python_codes/fonctions_step1_step2.py

```diff
@@ -19,37 +19,36 @@ def drop_punctuation(word):
     word = word.translate(str.maketrans('','',string.punctuation))
     return word.lower()
 
-emis['teacher_name'] = (emis['teacher_name'].apply(drop_digits)).apply(drop_punctuation)
-emis['teacher_surname'] = (emis['teacher_surname'].apply(drop_digits)).apply(drop_punctuation)
-payroll['Surname'] = (payroll['Surname'].apply(drop_digits)).apply(drop_punctuation)
-payroll['First name'] = (payroll['First name'].apply(drop_digits)).apply(drop_punctuation)
+emis['clean_teacher_name'] = (emis['teacher_name'].apply(drop_digits)).apply(drop_punctuation)
+emis['clean_teacher_surname'] = (emis['teacher_surname'].apply(drop_digits)).apply(drop_punctuation)
+payroll['clean_Surname'] = (payroll['Surname'].apply(drop_digits)).apply(drop_punctuation)
+payroll['clean_First name'] = (payroll['First name'].apply(drop_digits)).apply(drop_punctuation)
```

Comment:
do we have to keep capital letters in new column names ? And maybe replace ` `  with `_` , what do you think ?